src/main/resources/page/js/home-page-loader.js

```diff
@@ -0,0 +1,19 @@
+class HomePageLoader {
+    constructor() { }
+
+    loadHomePage() {
```

Comment:
If the name of the method is "load", this can be enough (HomePageLoader#load)

Reply:
I will rename it!